integration/index.ts

```diff
@@ -187,6 +197,13 @@ const grill = {
         )
     }
 
+    if (mergedConfig.customTexts) {
+      query.set(
+        'customTexts',
```

Comment:
I forget, that url size is limited and it is dangerous send lot of into in url, cna you research it, maybe better start using postMessage istead